native/cocos/editor-support/spine/TransformConstraint.h

```diff
@@ -73,11 +73,13 @@ class SP_API TransformConstraint : public Updatable {
     float getShearMix();
     void setShearMix(float inValue);
 
-    bool isActive();
+    virtual bool isActive();
```

Comment:
Why need virtual?

Reply:
Just the ensure that `isActive/setActive` is also virtual functions.
In fact, using the `override` suffix may be better, but I think spine runtime may support older compiler that only support c++98 standard which doesn't support `override` keyword.

In C++11 or later, you would get a warning like:
<img width="507" alt="企业微信截图_29906fd4-2212-4aef-a7b3-7203d47f738e" src="https://github.com/user-attachments/assets/769bfc26-e3ea-4559-9b58-f02c9d7ee45f">

I think it's a best practice of declaring a function with `virtual prefix`  or `override suffix` that override super's virtual function. It will be more explicit.

Reply:
Got it.